package-scripts.js

```diff
@@ -7,11 +7,12 @@ module.exports = {
     build: {
       default: {
         description: 'Runs all the build scripts in parallel',
-        script: 'p-s -p eslint,build.home,build.episodes,build.guests,build.deals,build.css,build.contributors,createJson',
+        script: 'p-s -p eslint,build.home,build.episodes,build.guests,build.linkstipspicks,build.deals,build.css,build.contributors,createJson',
       },
       home: 'babel-node generate/home',
       episodes: './scripts/build-episodes.sh',
       guests: 'rimraf guests && mkdir guests && babel-node generate/guests',
+      linkstipspicks: 'rimraf links-tips-picks && mkdir links-tips-picks && babel-node generate/links-tips-picks',
```

Comment:
could we call this `links-tips-picks`? Otherwise it all just kind of runs together :-)